marklogic-data-hub-central/ui/src/pages/Detail.tsx

```diff
@@ -456,6 +578,17 @@ const Detail: React.FC<Props> = ({history, location}) => {
                       </span>
                     </HCTooltip>}>
                   </Tab>
+                  {hasRelations && <Tab eventKey="relationships" id="relationships" data-cy="relationships-view" tabClassName={`${styles.tabActive} ${selected === "relationships" && styles.active}`} title={
+                    <HCTooltip id="relationships-tooltip" placement="top" text={"Show the relationships"}>
+                      <span>
+                        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I see we have added the ontology SVG file under /assets so let's import that and use it as a source in an <img> tag perhaps, instead of using the whole SVG code here.